akka-stream-testkit/src/test/scala/akka/stream/testkit/ChainSetup.scala

```diff
@@ -22,8 +22,8 @@ class ChainSetup[In, Out](
            toPublisher: (Source[Out, _], ActorFlowMaterializer) ⇒ Publisher[Out])(implicit system: ActorSystem) =
```

Comment:
should ChainSetup be public?
I think it is too advanced for the casual user, `TestPublisher.probe` is in general very easy to use in the wrong way.